.github/workflows/release.yml

```diff
@@ -42,6 +43,30 @@ jobs:
       - name: Determine version
         run: echo "wasmcloud_host_version=$(grep '@app_vsn "' ${{env.working-directory}}/mix.exs | cut -d '"' -f2)" > $GITHUB_ENV
 
+      # Install erlang/OTP and elixir
+      - name: Install erlang and elixir
+        uses: erlef/setup-beam@v1
+        with:
+          otp-version: "=24"
+          elixir-version: "1.12"
+          install-hex: true
+          install-rebar: true
+
+      - name: Install Mix Dependencies
+        working-directory: ${{env.working-directory}}
+        run: |
+          mix local.rebar --force 
+          mix local.hex --force
+          mix deps.get
+
+      - name: Compile Phoenix Assets
+        working-directory: ${{env.working-directory}}
+        # esbuild + dart sass are only included in the dev runtime environment
+        env:
+          MIX_ENV: dev
```

Comment:
Just to double check, doing this with `dev` isn't going to screw up `prod` stuff down the line?

Reply:
That's right, we just only include the SASS compiler in the development environment to keep the prod env as minimal as possible

Reply:
Yeah I used the localized `env` option which should only apply to that one step

Reply:
Yes, got that too. Obviously I am not explaining myself well 😅 . 

So the concern here is: Will debug/dev on these steps end up including large or unneeded resources in the final release tarball (even though the rest are using `prod`)?

Reply:
Ah okay I think I understand better. So this step does end up doing a bit of unnecessary computation by compiling dependencies for dev, but the only assets that will carry over to the release tarball are the minified CSS files that are compiled. Put another way, running this step in dev mode is just a tool to keep the sass dependency out of the release binary and keep it only in the dev environment.

Reply:
Apologies for merging before this is resolved